src/Pim/PimMain.tsx

```diff
@@ -34,6 +36,8 @@ interface PropsType {
   location?: Location;
   history?: History;
   theme: Theme;
+  collectionsTaskList: EteSync.CollectionInfo[];
+  onItemSave: (item: PimType, journalUid: string, originalContact?: PimType) => void;
```

Comment:
Could you please fix the naming here? It should be `originalTask`, or better yet, just `originalItem`.